### ISSUE TITLE
How to adjust the color of `Code is unreachable` and report type `information`?

### ISSUE THREAD

**Issue by timsu92** [fannheyward/coc-pyright#813]
**What's the output of `:CocCommand pyright.version`**
```
[coc.nvim] coc-pyright 1.1.269 with Pyright 1.1.269
```

**What's the output of `:CocCommand workspace.showOutput Pyright`**
```
Workspace: /home/tim
Using python from /home/tim/.local/share/virtualenvs/tim-ys9FBRsU/bin/python

[Warn  - 下午9:48:35] Ignored rootPath /home/tim of client "pyright"
[Warn  - 下午9:48:35] No valid workspaceFolder exists
[Info  - 下午9:48:35] Pyright language server 1.1.269 starting
[Info  - 下午9:48:35] Server root directory: /home/tim/.config/coc/extensions/node_modules/coc-pyright/node_modules/pyright/dist/
[Info  - 下午9:48:35] Starting service instance "<default>"
[Info  - 下午9:48:35] No pyproject.toml file found.
[Info  - 下午9:48:35] Setting pythonPath for service "<default>": "/home/tim/.local/share/virtualenvs/tim-ys9FBRsU/bin/python"
[Warn  - 下午9:48:35] stubPath typings is not a valid directory.
[Info  - 下午9:48:35] Assuming Python version 3.10
[Info  - 下午9:48:35] Assuming Python platform Linux
[Info  - 下午9:48:35] Searching for source files
[Info  - 下午9:48:35] No source files found.
```

## Problem

I'm using [monokai](https://github.com/crusoexia/vim-monokai) as my colorscheme, and I have this in my `~/.vim/coc-settings.json`:
```json
{
    "python.analysis.diagnosticSeverityOverrides": {
        "reportUnusedImport": "information",
        "reportDuplicateImport": "warning"
    }
}
```

I typed a short Python code showing like this in vim:
![image](https://user-images.githubusercontent.com/33785401/188169651-acff8b3b-3daa-4fbd-a116-b35920fa7871.png)
We can see that line 2 is reported as "information", line 3 is "error" and line 4 is warning.
However, "information" is grabbing our eyes rather than warning nor error.

## Things I considered

I thought it might be my color scheme's problem. However, no matter which I use, the color of `Code is unreachable` and report type `information` remain the same.

## Suggestion

I found there are [dedicated colors for coc in monokai](https://github.com/crusoexia/vim-monokai/blob/master/colors/monokai.vim#L236-L251).
I think I might be better to use CocInfo for coloring report type `information` and [comment](https://github.com/crusoexia/vim-monokai/blob/master/colors/monokai.vim#L210) for `Code is unreachable`

**Comment by fannheyward** [comment]
Change `CocInfoHighlight` and try again, I can't reproduce this with `dracula` scheme.

<img width="246" alt="截屏2022-09-03 16 11 11" src="https://user-images.githubusercontent.com/345274/188262150-fafa3ae3-c41f-4fd1-aa6c-74cbae0d7062.png">

**Comment by timsu92** [comment]
Sorry I forget to say that the cursor can't be on the same line. Otherwise, it's truly not able to see that😅

**Comment by timsu92** [comment]
> Change CocInfoHighlight and try again

OK, I changed it and found that this plugin is making the background gray.
I checked this by changing `CocInfoHighlight`. Plus, another plugin 
called [coc-spell-checker] use this color as well. I know because changing `CocInfoHighlight` changes the color of misspelled words.

Nevertheless, different `CocInfoHighlight` only gets different background of report type `information`.
I'm wondering whether `Code is unreachable` and report type `information` is colored in different ways, and the latter itself uses two methods as well?

[coc-spell-checker]: https://github.com/iamcco/coc-spell-checker
